### PR TITLE
Jdp200202 15

### PR DESCRIPTION
**Pull request by EmiCoder**

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+
+    @Column(name="first_name")
+    private String first_name;
+
+    @Column(name="last_name")
+    private String last_name;
```
Comment (mateusz58) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]: Camel case standard must be followed. https://www.geeksforgeeks.org/java-naming-conventions/
```suggestion
    private String lastName;
```

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Entity
+@Table(name="users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+
+    @Column(name="first_name")
+    private String first_name;
```
Comment (mateusz58) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]:
```suggestion
    private String firstName;
```

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 3]
```diff
@@ -22,11 +22,11 @@
     @GeneratedValue(strategy= GenerationType.AUTO)
     private Long id;
 
-    @Column(name="first_name")
-    private String first_name;
+    @Column(name="firstName")
+    private String firstName;
```
Comment (g-chmura) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]:
```suggestion
    @Column(name="first_name")
    private String firstName;
```

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 4]
```diff
@@ -22,11 +22,11 @@
...
 
-    @Column(name="last_name")
-    private String last_name;
+    @Column(name="lastName")
+    private String lastName;
```
Comment (g-chmura) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]:
```suggestion
    @Column(name="last_name")
    private String lastName;
```

**Pull request merged**